public_html/wp-content/plugins/wcpt/wcpt-event/class-event-admin.php

```diff
@@ -455,6 +456,7 @@ public function metabox_save( $post_id, $post, $verify_nonce = true ) {
 		}
 
 		// Don't add/remove meta on trash, untrash, restore, etc.
+		// phpcs:ignore WordPress.Security.NonceVerification.Missing
```

Comment:
That sniff has so many false positives, I tempted to just say we should disable it.

It is a security one, though, which makes me leery of that. I hate adding cluttering the code with `phpcs:ignore`, but maybe in this case that's a reasonable compromise?

What do y'all think? @vedanshujain , @coreymckrill

Reply:
lets keep it, because as you say, it sound like a reasonable compromise given that this affects security. we can revisit this discussion later if this gets too annoying.